### PR TITLE
Fix/add sass to storybook

### PR DESCRIPTION
**Pull request by arselt**
Summary 💡
Added scss/sass support to storybook + global configuration
#161

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,4 +33,4 @@
     "url": "https://github.com/Platzi-Master-C9/booking-ui/issues"
   },
   "homepage": "https://github.com/Platzi-Master-C9/booking-ui#readme"
-}
+}
```
Comment (elyager) [inline package.json:36]: It's missing the EOF

File: yarn.lock [hunk 2]
```diff
@@ -1,16191 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (elyager) [inline yarn.lock:1]: yarn.lock file should exists in the repo and is not meant to be edited manually, please don't delete this file

**Review by elyager — CHANGES_REQUESTED** [review]

File: yarn.lock (hunk 2 above)
Comment (arselt) [reply yarn.lock]: Regenerated the yarn.lock and merged it check [3eabd74](https://github.com/Platzi-Master-C9/booking-ui/pull/190/commits/3eabd740f6403543c9843a00a8492f0023891a98)

**Review by elyager — APPROVED** [review]
LGTM

**Pull request merged**